tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -6,72 +6,76 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/credentials"
-	sessionaws "github.com/aws/aws-sdk-go/aws/session"
+	awssession "github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/aws/aws-sdk-go/service/iam"
-	v1 "github.com/openshift/api/config/v1"
-	client "github.com/openshift/client-go/config/clientset/versioned"
 	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/cloudprovider"
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/openshift"
 )
 
-// svc is a AWS specific struct which contains service information needed to communicate with AWS and the existing cluster
-// svc should remain private since secretes should not be exposed
-type svc struct {
-	// service account for AWS EC2
+// cloudProviderInfo is an provider specific struct which contains clients for EC2, IAM,
+// and the existing OpenShift cluster that is running on EC2.
+// This is an implementation of the Cloud interface.
+type cloudProviderInfo struct {
+	// A client for EC2.
 	EC2 *ec2.EC2
-	// service account for AWS IAM
+	// A client for IAM.
 	IAM *iam.IAM
-	// client of the existing cluster
-	clientset *client.Clientset
-	// infraID is the readable unique infrastructure name generated by openshift-install for new infrastructures
-	infraID *string
-	// dir is the path to where created instance and security group information is stored
-	dir *string
+	// clientset is the client of the existing OpenShift cluster.
+	Clientset *openshift.Client
```

Comment:
Please follow go naming conventions. Moreover how does one know that `clientset` is the OpenShift client in the places you are using it. I suggest calling it `openShiftClient`. This applies to all places you are using `clientset`.